src/components/CDKType/CDKType.tsx

```diff
@@ -9,10 +9,11 @@ import { CDKType, CDKTYPE_RENDER_MAP } from "../../constants/constructs";
 
 interface CDKTypeIconProps extends ImageProps {
   name?: CDKType;
+  majorVersion?: number;
 }
 
 export const CDKTypeIcon = forwardRef<CDKTypeIconProps, "img">(
-  ({ name, ...props }, ref) => {
+  ({ name, majorVersion, ...props }, ref) => {
```

Comment:
Curious about this addition here

Reply:
This is an unrelated bug fix - I saw a console error that said (paraphrasing) "majorVersion can't be passed as a prop to this IMG element" and I figured out it's because it was getting grouped into `...props` here.